aws_batch/Dockerfile

```diff
@@ -61,6 +61,9 @@ RUN wget http://launchpadlibrarian.net/162265652/fastx-toolkit_0.0.14-1_amd64.de
 RUN gdebi --non-interactive libgtextutils0_0.7-1_amd64.deb
 RUN gdebi --non-interactive fastx-toolkit_0.0.14-1_amd64.deb
 
+# Install IDseq pipeline package
+RUN pip install git+https://github.com/chanzuckerberg/idseq-pipeline.git
```

Comment:
I actually don't think we should put this into the Docker image. We should install the package as part of the batch command. Updating dockerimage is kinda painful.

Reply:
painful how? we can get it automated